### DISCUSSION TITLE
Tab switching across panels.

### DISCUSSION THREAD

**Discussion by donjan** [zed-industries/zed#17938] [Feature Requests]
I can successfully switch through my tabs with
```
      "ctrl-shift-tab": "pane::ActivatePrevItem",
      "ctrl-tab": "pane::ActivateNextItem",
```
But with a left-right split pane, this cycles through the tabs of only one pane. In VSCode, it jumps to the next pane once the end is reached.
I realise that Zed's more flexible 2-dimensional pane system makes it trickier to decide which pane to switch next to. But I don't ever use top-bottom split panes. Moving to the next tab in the other pane requires either using the mouse or a different shortcut for `["workspace::ActivatePaneInDirection", "Right"]`, which is a regression in my workflow.
I've tried "overbinding" both actions to `"ctrl-tab"`, but only the first one takes.
Can anything be done about this?

**Comment by lightiverson** [comment]
I was thinking about using `context` to determine I am at the last tab and then overwriting the shortcut for next tab to next pane. But it seems this sort of `context` is too complicated.

I would also like to see the VSCode behaviour of jumping panes once the end is reached.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
